pkg/core/contracts/pools/BasePool.sol

```diff
@@ -453,7 +453,7 @@ abstract contract BasePool is IBasePool, BasePoolAuthorization, BalancerPoolToke
      */
     function _addSwapFeeAmount(uint256 amount) internal view returns (uint256) {
         // This returns amount + fee amount, so we round up (favoring a higher fee amount).
-        return amount.divUp(_swapFeePercentage.complement());
+        return amount.divUp(FixedPoint.ONE.sub(_swapFeePercentage));
```

Comment:
This is just more clear, but it doesn't change the behavior, right?

Reply:
We did change the behavior, but the swap fee pct is guaranteed to be lower than or equal to 50%